hooks/pre/10-reset_feature.rb

```diff
@@ -19,8 +19,9 @@ def foreman_installed?
 end
 
 def stop_services
-  Kafo::KafoConfigure.logger.info 'Ensuring services httpd and foreman-tasks are stopped.'
-  Kafo::Helpers.execute('service httpd stop && service foreman-tasks stop')
+  Kafo::KafoConfigure.logger.info 'Stopping services'
+  Kafo::Helpers.execute('katello-service stop')
```

Comment:
The katello-service script supports a way to exclude services that could be used instead of starting postgresql in the next line.